areas-of-d-usage.dd

```diff
@@ -196,51 +196,48 @@ $(AREA_SECTION3 $(LNAME2 gpu, GPU Programming), $(ARTICLE_FA_ICON flash),
+    analysis. For example, $(LINK2 http://ddili.org/ders/d.en/ranges.html, D's ranges)
     are a natural fit for pipes and data streams.
-    One can start from a small D script with a small subset to test
-    some analysis and easily switch to the full set by enabling optimizations
+    One can start from a small D script with a small subset of data for
+    an analysis, and easily switch to the full set by enabling optimizations
     for superior performance.
-    Read Adroll's (Big data marketing company) testimonial
+    Read Adroll's testimonial
     $(HTTP tech.adroll.com/blog/data/2014/11/17/d-is-for-data-science.html, "D is for Data Science")
-    which explains why D is a keystone language for their critical infrastructure.
+    on why D is a keystone language for their critical infrastructure.
     eBay recently open-sourced their internal $(HTTPS github.com/eBay/tsv-utils-dlang, data processing utilities)
-    that are used in their large-scale data mining environment.
+    used in their large-scale data mining environment.
 ))
 
 $(SECTION2 $(LNAME2 academia, Academia),
 
 $(AREA_SECTION3 $(LNAME2 teaching, Teaching), $(ARTICLE_FA_ICON graduation-cap),
 
     $(P D's multi-paradigm (imperative, structured, object oriented, generic, functional programming
-        purity, and even assembly) approach allows to teach in one language and gradually
-        explain new features without needing to switch to a different language.
+        purity, and even assembly) approach allows teaching in one language and gradually
+        explaining new features without needing to switch to a different language.
     )
-    $(P At universities Java is commonly taught because of its memory safety.
-        D has a memory-safe subset $(I $(LINK2 $(ROOT_DIR)/safed.html, SafeD))
-        which safety and ease of use is comparable to Java while still being efficient.
-        Beginners will also highly profit from the Garbage Collection in D which helps
-        them to write simple code without any memory management knowledge.
-        However more sophisticated memory management can easily be done in an advanced
+    $(P At universities, Java is commonly taught because of its memory safety.
+        $(LINK2 $(ROOT_DIR)safed.html, SafeD), a memory-safe subset of D, provides
+        safety and ease of use comparable to Java while remaining efficient.
+        Beginners will also benefit from garbage collection, which helps
+        write simple code without requiring any knowledge of explicit memory management.
+        $(LINK2 $(ROOT_DIR)phobos/std_experimental_allocator.html, More sophisticated memory management) can be introduced in an advanced
         learning stage.
     )
-    $(P Moreover D's blazingly fast runtime allows it to be used as a high level,
+    $(P D's blazingly fast compilation allows it to be used as a high level,
         productive scripting language - just like Python, but with the
```

Comment:
Killed the whole Python reference. The fewer potshots we take at other languages specifically, the better.